.gitlint

```diff
@@ -0,0 +1,13 @@
+# https://jorisroovers.com/gitlint/latest/configuration/
+
+[general]
+ignore=body-is-missing
+
+[title-max-length]
+line-length=80
+
+[title-min-length]
+min-length=5
+
+[title-match-regex]
+regex=^(build|ci|docs|feat|fix|perf|refactor|test|chore)
```

Comment:
```suggestion
regex=^(build|ci|docs|feat|fix|perf|refactor|test|chore|release)
```
Does it make sense to include also the `release` prefix ?

Reply:
Removed gitlint in https://github.com/elastic/oblt-actions/pull/1/commits/68d6e7c090723ea3578da8a53ed13550f32ecab9 because I'll be adding a label based version and release flow with release-drafter